Libraries/LibGUI/GAbstractView.h

```diff
@@ -44,7 +44,7 @@ class GAbstractView : public GScrollableWidget {
 
     GModelSelection& selection() { return m_selection; }
     const GModelSelection& selection() const { return m_selection; }
-    void select_all();
+    virtual void select_all();
```

Comment:
Does the empty default implemention make sense? I would say this should be a pure virtual method.